src/refactor_nrepl/artifacts.clj

```diff
@@ -101,6 +101,14 @@
                    (map #(vector (str group-id "/" %) nil))))
             group-ids)))
 
+(defn get-clojars-versions!
+  "Fetches all the versions of particular artifact from Clojars."
+  [for-artifact]
```

Comment:
I'd name this just `artifact`.

Reply:
Me too, I followed the convention of the other function. Will update!